packages/amplication-client/src/Application/OverviewSecondaryTile/OverviewSecondaryTile.scss

```
@@ -0,0 +1,61 @@
+@import "../../style/index.scss";
+
+.overview-secondary-tile {
+  padding: var(--large-spacing);
+  min-height: 200px;
+  background-color: var(--black2);
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  position: relative;
+  overflow: hidden;
+
+  &:hover {
+    box-shadow: 0px 24px 25px -8px #090b1466;
+    transition: all 1s ease;
+    border: 1px solid var(--black30);
```

please use parameter
```suggestion
    border: $border-black30;
```